tests/rptest/tests/tx_admin_api_test.py

```diff
@@ -244,3 +245,56 @@ def test_all_transactions(self):
                 expected_partitions[topic.name]) == topic.partition_count
             for partition in range(topic.partition_count):
                 assert partition in expected_partitions[topic.name]
+
+    @cluster(num_nodes=3)
+    def test_delete_topic_from_tx(self):
```

Comment:
also please add tests for the boundary/error conditions that you check for on the REST interface